scripts/create_testenv.sh

```diff
@@ -24,13 +24,15 @@ ENVNAME="${ENVNAME:-testenv}"         # if no ENVNAME is specified, use testenv
 if [ -z ${GLOBAL} ]; then
   if conda env list | grep -q ${ENVNAME}; then
     echo "Environment ${ENVNAME} already exists, keeping up to date"
+    source activate ${ENVNAME}
```

Comment:
I think new style is `conda activate` (or mamba).

Reply:
That doesn't work right now, but according to [this](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/use-conda-with-travis-ci.html#the-travis-yml-file) and [this](https://github.com/conda/conda/issues/7980) it should be fixed by sourcing the conda functions again:
```
source $CONDA_PREFIX/etc/profile.d/conda.sh
```
trying...

Reply:
If it's brittle probably best to keep it the way you had then.

Reply:
Well, it is now the way the conda docs specify it...
But I can of course change it back.